rfcs/0018-glides-trainsheet-events.md

```diff
@@ -258,71 +273,87 @@ All events should have their schema documented in the [mbta/schemas](https://git
 ### Compatibility
 Changes to the events can happen in one of two ways:
 - backwards-compatible changes (such as adding a new field with a default value) where old consumers can still interpret new events can be made while keeping the same event type. 
-- incompatible-changes require creating a new event type. Some options as described in [RFC 4](https://github.com/mbta/technology-docs/blob/main/rfcs/accepted/0004-socket-proxy-ocs-cloudevents.md) are `<type>.<date>` (adding the date of the new event) or `<type>.v2` (creating a new version). Each version should be documented separately in the [mbta/schemas](https://github.com/mbta/schemas) GitHub repository.
+- incompatible-changes require creating a new event type, `<type>.v2` (creating a new version). Each version should be documented separately in the [mbta/schemas](https://github.com/mbta/schemas) GitHub repository.
```

Comment:
Does Glides have to keep publishing old `v1` events in parallel, or are consumers expected to update to handle both `v1` and `v2` before Glides switches if they don't want to miss events?

Reply:
I can include this in the updated compatibility section (see also [my comment](https://github.com/mbta/technology-docs/pull/18#discussion_r1094691209) to Melody). I think this depends on the Glides consumers. If all of the consumers at a particular time can support the latest version, then Glides can drop sending the old version of the events. But as long as there are consumers who do not support the new events, then Glides SHOULD (MUST?) continue to send the old events.